app/components/ui/set-up-domain/select-blog-type/index.js

```diff
@@ -53,17 +53,10 @@ class SelectBlogType extends Component {
 			<div className={ styles.domainSetup }>
 				<DocumentTitle title={ i18n.translate( 'Set up domain' ) } />
 
-				<div className={ styles.headerContainer }>
-					<div className={ styles.header }>
-						<h1 className={ styles.headerText }>
-							<span className={ styles.setUpLabel }>
-								{ i18n.translate( 'Setup: ' ) }
-							</span>
-							{ i18n.translate( 'Tell us about your blog.' ) }
-						</h1>
-						<ProgressBar progress={ 10 } />
-					</div>
-				</div>
+				<ProgressHeader
+					content={ i18n.translate( 'Tell us about your blog.' ) }
+					progress={ 10 }
+				/>
```

Comment:
I think this needs to be updated such that this component has children instead of a `content` prop, so that the text appears:

![screen shot 2017-02-07 at 10 17 09 am](https://cloud.githubusercontent.com/assets/1130674/22705199/7c9f4c16-ed1f-11e6-84b2-8e6916b2c92c.png)